From 29d98d496733bb593b0f1165e82e14a83f19eec0 Mon Sep 17 00:00:00 2001
From: Huy Anh Le <hule@amperecomputing.com>
Date: Thu, 15 May 2025 09:27:19 +0700
Subject: [PATCH] configurations: mtjade: update Min Max values

Update Min Max values calculated follow value per
hardware recommendation

Tested:
 1. Boot up and login to the OpenBmc console.
 2. Execution commands to get IPMI SEL and redfish
    SEL of threshold event
    $ ipmitool sensor get <Sensor_Name>
    $ <bmc_ip>/redfish/v1/Systems/system/LogServices/EventLog/Entries
 3. Check value and compare value of IPMI and Redfish

Upstream-Status: Pending

Signed-off-by: Huy Anh Le <hule@amperecomputing.com>
---
 configurations/mtjade.json | 112 +++++++++++++++++++++++++++++++++----
 1 file changed, 100 insertions(+), 12 deletions(-)

diff --git a/configurations/mtjade.json b/configurations/mtjade.json
index 8a50c63..8a41bba 100644
--- a/configurations/mtjade.json
+++ b/configurations/mtjade.json
@@ -3,6 +3,8 @@
         "Exposes": [
             {
                 "Index": 28,
+                "MaxValue": 5.1,
+                "MinValue": 0,
                 "Name": "3V3_SB",
                 "PollRate": 1,
                 "ScaleFactor": 0.3333,
@@ -31,6 +33,8 @@
                     }
                 ],
                 "Index": 29,
+                "MaxValue": 4.59,
+                "MinValue": 0,
                 "Name": "VBAT_RTC",
                 "PollRate": 10,
                 "ScaleFactor": 0.3333,
@@ -53,6 +57,8 @@
             {
                 "CPURequired": 0,
                 "Index": 0,
+                "MaxValue": 1.275,
+                "MinValue": 0,
                 "Name": "S0_0V8_VDDC_SOC",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -76,6 +82,8 @@
             {
                 "CPURequired": 1,
                 "Index": 1,
+                "MaxValue": 1.275,
+                "MinValue": 0,
                 "Name": "S1_0V8_VDDC_SOC",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -99,6 +107,8 @@
             {
                 "CPURequired": 0,
                 "Index": 2,
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S0_0V75_PCP",
                 "PollRate": 1,
                 "PowerState": "On",
@@ -122,6 +132,8 @@
             {
                 "CPURequired": 1,
                 "Index": 3,
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S1_0V75_PCP",
                 "PollRate": 1,
                 "PowerState": "On",
@@ -145,6 +157,8 @@
             {
                 "CPURequired": 0,
                 "Index": 4,
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S0_1V2_VDDQ0123",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -168,6 +182,8 @@
             {
                 "CPURequired": 1,
                 "Index": 5,
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S1_1V2_VDDQ0123",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -191,6 +207,8 @@
             {
                 "CPURequired": 0,
                 "Index": 6,
+                "MaxValue": 1.02,
+                "MinValue": 0,
                 "Name": "S0_0V6_VTT0123",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -214,6 +232,8 @@
             {
                 "CPURequired": 1,
                 "Index": 7,
+                "MaxValue": 1.02,
+                "MinValue": 0,
                 "Name": "S1_0V6_VTT0123",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -237,6 +257,8 @@
             {
                 "CPURequired": 0,
                 "Index": 8,
+                "MaxValue": 3.825,
+                "MinValue": 0,
                 "Name": "S0_2V5_VPP0123",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -260,6 +282,8 @@
             {
                 "CPURequired": 1,
                 "Index": 9,
+                "MaxValue": 3.825,
+                "MinValue": 0,
                 "Name": "S1_2V5_VPP0123",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -283,6 +307,8 @@
             {
                 "CPURequired": 0,
                 "Index": 10,
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S0_1V2_VDDQ4567",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -306,6 +332,8 @@
             {
                 "CPURequired": 1,
                 "Index": 11,
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S1_1V2_VDDQ4567",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -329,6 +357,8 @@
             {
                 "CPURequired": 0,
                 "Index": 12,
+                "MaxValue": 1.02,
+                "MinValue": 0,
                 "Name": "S0_0V6_VTT4567",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -352,6 +382,8 @@
             {
                 "CPURequired": 1,
                 "Index": 13,
+                "MaxValue": 1.02,
+                "MinValue": 0,
                 "Name": "S1_0V6_VTT4567",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -375,6 +407,8 @@
             {
                 "CPURequired": 0,
                 "Index": 14,
+                "MaxValue": 3.825,
+                "MinValue": 0,
                 "Name": "S0_2V5_VPP4567",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -398,6 +432,8 @@
             {
                 "CPURequired": 1,
                 "Index": 15,
+                "MaxValue": 3.825,
+                "MinValue": 0,
                 "Name": "S1_2V5_VPP4567",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -421,6 +457,8 @@
             {
                 "CPURequired": 0,
                 "Index": 16,
+                "MaxValue": 1.275,
+                "MinValue": 0,
                 "Name": "S0_0V85_RCA",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -444,6 +482,8 @@
             {
                 "CPURequired": 1,
                 "Index": 17,
+                "MaxValue": 1.275,
+                "MinValue": 0,
                 "Name": "S1_0V85_RCA",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -467,6 +507,8 @@
             {
                 "CPURequired": 0,
                 "Index": 18,
+                "MaxValue": 2.805,
+                "MinValue": 0,
                 "Name": "S0_1V8_VDDH",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -490,6 +532,8 @@
             {
                 "CPURequired": 1,
                 "Index": 19,
+                "MaxValue": 2.805,
+                "MinValue": 0,
                 "Name": "S1_1V8_VDDH",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -513,6 +557,8 @@
             {
                 "CPURequired": 0,
                 "Index": 20,
+                "MaxValue": 2.295,
+                "MinValue": 0,
                 "Name": "S0_1V5_VDDH",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -536,6 +582,8 @@
             {
                 "CPURequired": 1,
                 "Index": 21,
+                "MaxValue": 2.295,
+                "MinValue": 0,
                 "Name": "S1_1V5_VDDH",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -559,6 +607,8 @@
             {
                 "CPURequired": 0,
                 "Index": 22,
+                "MaxValue": 2.805,
+                "MinValue": 0,
                 "Name": "S0_1V8_SOC",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -582,6 +632,8 @@
             {
                 "CPURequired": 1,
                 "Index": 23,
+                "MaxValue": 2.805,
+                "MinValue": 0,
                 "Name": "S1_1V8_SOC",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -605,6 +657,8 @@
             {
                 "CPURequired": 0,
                 "Index": 24,
+                "MaxValue": 2.805,
+                "MinValue": 0,
                 "Name": "S0_1V8_PCP",
                 "PollRate": 1,
                 "PowerState": "On",
@@ -628,6 +682,8 @@
             {
                 "CPURequired": 1,
                 "Index": 25,
+                "MaxValue": 2.805,
+                "MinValue": 0,
                 "Name": "S1_1V8_PCP",
                 "PollRate": 1,
                 "PowerState": "On",
@@ -651,6 +707,8 @@
             {
                 "CPURequired": 0,
                 "Index": 26,
+                "MaxValue": 5.1,
+                "MinValue": 0,
                 "Name": "S0_3V3_SOC",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -674,6 +732,8 @@
             {
                 "CPURequired": 1,
                 "Index": 27,
+                "MaxValue": 5.1,
+                "MinValue": 0,
                 "Name": "S1_3V3_SOC",
                 "PollRate": 1,
                 "PowerState": "ChassisOn",
@@ -704,7 +764,7 @@
                     ]
                 },
                 "Index": 4,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN3_1",
                 "PowerState": "On",
                 "Thresholds": [
@@ -733,7 +793,7 @@
                     ]
                 },
                 "Index": 5,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN3_2",
                 "PowerState": "On",
                 "Thresholds": [
@@ -762,7 +822,7 @@
                     ]
                 },
                 "Index": 6,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN4_1",
                 "PowerState": "On",
                 "Thresholds": [
@@ -791,7 +851,7 @@
                     ]
                 },
                 "Index": 7,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN4_2",
                 "PowerState": "On",
                 "Thresholds": [
@@ -820,7 +880,7 @@
                     ]
                 },
                 "Index": 8,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN5_1",
                 "PowerState": "On",
                 "Thresholds": [
@@ -849,7 +909,7 @@
                     ]
                 },
                 "Index": 9,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN5_2",
                 "PowerState": "On",
                 "Thresholds": [
@@ -878,7 +938,7 @@
                     ]
                 },
                 "Index": 10,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN6_1",
                 "PowerState": "On",
                 "Thresholds": [
@@ -907,7 +967,7 @@
                     ]
                 },
                 "Index": 11,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN6_2",
                 "PowerState": "On",
                 "Thresholds": [
@@ -936,7 +996,7 @@
                     ]
                 },
                 "Index": 12,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN7_1",
                 "PowerState": "On",
                 "Thresholds": [
@@ -965,7 +1025,7 @@
                     ]
                 },
                 "Index": 13,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN7_2",
                 "PowerState": "On",
                 "Thresholds": [
@@ -994,7 +1054,7 @@
                     ]
                 },
                 "Index": 14,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN8_1",
                 "PowerState": "On",
                 "Thresholds": [
@@ -1023,7 +1083,7 @@
                     ]
                 },
                 "Index": 15,
-                "MaxReading": 23100,
+                "MaxReading": 25500,
                 "Name": "FAN8_2",
                 "PowerState": "On",
                 "Thresholds": [
@@ -1057,6 +1117,8 @@
                 "Type": "MB Upstream Port"
             },
             {
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "S0_DIMMG0_Temp",
                 "Sensors": [
                     "S0_DIMM_CH0_Temp",
@@ -1076,6 +1138,8 @@
                 "Units": "DegreesC"
             },
             {
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "S1_DIMMG0_Temp",
                 "Sensors": [
                     "S1_DIMM_CH0_Temp",
@@ -1133,6 +1197,8 @@
                 "Units": "DegreesC"
             },
             {
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S0_DIMM_DDR_Volt",
                 "Sensors": [
                     "S0_DIMM_VR1_Volt",
@@ -1142,6 +1208,8 @@
                 "Units": "Volts"
             },
             {
+                "MaxValue": 1.785,
+                "MinValue": 0,
                 "Name": "S1_DIMM_DDR_Volt",
                 "Sensors": [
                     "S1_DIMM_VR1_Volt",
@@ -1349,6 +1417,8 @@
             {
                 "Address": "0x28",
                 "Bus": 3,
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "TS1_Temp",
                 "Thresholds": [
                     {
@@ -1363,6 +1433,8 @@
             {
                 "Address": "0x29",
                 "Bus": 3,
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "TS2_Temp",
                 "Thresholds": [
                     {
@@ -1377,6 +1449,8 @@
             {
                 "Address": "0x2a",
                 "Bus": 3,
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "TS3_Temp",
                 "Thresholds": [
                     {
@@ -1391,6 +1465,8 @@
             {
                 "Address": "0x2b",
                 "Bus": 3,
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "TS4_Temp",
                 "Thresholds": [
                     {
@@ -1405,6 +1481,8 @@
             {
                 "Address": "0x2c",
                 "Bus": 3,
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "TS5_Temp",
                 "Thresholds": [
                     {
@@ -1419,6 +1497,8 @@
             {
                 "Address": "0x2d",
                 "Bus": 3,
+                "MaxValue": 255,
+                "MinValue": 0,
                 "Name": "TS6_Temp",
                 "Thresholds": [
                     {
@@ -1463,11 +1543,15 @@
                 ],
                 "Type": "pmbus",
                 "fan1_Name": "PSU0_fan1",
+                "fan1_Max": 25500,
+                "fan1_Min": 0,
                 "iin_Name": "PSU0_IINPUT",
                 "iout1_Name": "PSU0_IOUTPUT",
                 "pin_Name": "PSU0_PINPUT",
                 "pout1_Name": "PSU0_POUTPUT",
                 "temp1_Name": "PSU0_TEMP",
+                "temp1_Max": 255,
+                "temp1_Min": 0,
                 "vin_Name": "PSU0_VINPUT",
                 "vout1_Name": "PSU0_VOUTPUT"
             },
@@ -1497,11 +1581,15 @@
                 ],
                 "Type": "pmbus",
                 "fan1_Name": "PSU1_fan1",
+                "fan1_Max": 25500,
+                "fan1_Min": 0,
                 "iin_Name": "PSU1_IINPUT",
                 "iout1_Name": "PSU1_IOUTPUT",
                 "pin_Name": "PSU1_PINPUT",
                 "pout1_Name": "PSU1_POUTPUT",
                 "temp1_Name": "PSU1_TEMP",
+                "temp1_Max": 255,
+                "temp1_Min": 0,
                 "vin_Name": "PSU1_VINPUT",
                 "vout1_Name": "PSU1_VOUTPUT"
             },
